content/post/session3-startupdeepdive.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Start-Up Deep Dive : Revenue Model and Building Sustainable Products from Kerala"
+date: 2023-01-25T00:55:13+05:30
+Description: "A highlight of the third session of Startup Deep Dive taken by Felix Josemon on the topic “Revenue Model”."
+authors:
+  - jesa-dileep-philip
+Tags: [ "startup","startupdeepdive","revenue-models"]
```

Comment:
please add "ets" tag here. Let's add it for all ETS posts to easily filter out later no?

Reply:
sure